### PR TITLE
Update developer guide (testing instructions for link-realted commands)

### PR DESCRIPTION
**Pull request by shwene**

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -1095,6 +1095,62 @@ testers are expected to do more *exploratory* testing.
 
 1. _{ more test cases … }_
 
+### Adding a link
+
+1. Adding a link to a module in Plannit
+
+    1. Prerequisites: Use the `list` command to display all existing modules (from the sample data) in Plannit.
```
Comment (cheeheng) [inline docs/DeveloperGuide.md:1102]:
```suggestion
    1. Prerequisite: Use the `list` command to display all existing modules (from the sample data) in Plannit.
```
Only one prerequisite, so singular form? Same for deleting and opening link.

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -1095,6 +1095,62 @@ testers are expected to do more *exploratory* testing.
...
+
+    1. Prerequisites:  Use the `list` command to display all existing modules (from the sample data) in Plannit.
+
+    1. Test case: `open-link m/CS2103T la/Team Repo`<br>
+       Expected:
+        * The link with the alias `Team Repo` is opened from the module with module code `CS2103T` into your default browser.
+
+    1. Test case: `open-link m/CS2103T la/hahahaha`<br>
+       Expected:
+        * No links is opened from the module with module code `CS2103T`.
```
Comment (cheeheng) [inline docs/DeveloperGuide.md:1148]:
```suggestion
        * No links are opened from the module with module code `CS2103T`.
```
Grammatical error? Same for delete link.

**Review by cheeheng — COMMENTED** [review]
LGTM except for grammatical error nitpicks (which count as documentation bug).

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (shwene) [reply docs/DeveloperGuide.md]: Thanks! Have applied the suggestions.

**Review by cheeheng — APPROVED** [review]
LGTM

**Review by teoyuqi — APPROVED** [review]
LGTM

**Pull request merged**